### ISSUE TITLE
Almost there!

### ISSUE THREAD

**Issue by Kmonk** [MegaPirateNG/ardupilot-mpng#33]
I'm running a HK AIO pro V2 with ArduCopter-MPNG V3.0.1 R2 on a F450 quad, I seem the now have everything loaded/configured/calibrated but still must be missing something.

At power up my simonk flashed esc's give a three tone signal and then after a pause give a single beep every three seconds or so and the motors wont respond to the throttle.

**Comment by smurfy** [comment]
- Is your radio calibriated correctly?
- Do you arm your copter?
- Are your esc's calibriated?

**Comment by Kmonk** [comment]
I was able to calibrate my radio with mission planer.
The esc's were calibrated individually after being flashed.
It shows in Mission planer that my quad is disarmed, I'm using a Futaba T8FG super which wont allow me to arm my quad by holding the yaw to the right with the throttle in it's lowest position.

**Comment by smurfy** [comment]
> It shows in Mission planer that my quad is disarmed, I'm using a Futaba T8FG super which wont allow me to arm my quad by holding the yaw to the right with the throttle in it's lowest position. 

Arming should be possible by moving the left stick to lower right.
It could take a couple of seconds.
Also there is a possibility that an error message will be displayed if you did not performed all calibrations needed.
(this prearm check can be disabled via parameters via mission planner)

Also i had to recalibriate my esc via auto apm calibriation. (my copter worked great with multiwii before)

**Comment by Kmonk** [comment]
Just changed over to a RCTimer Crius AIO pro V2 which armed and worked without a problem.

I would rather use the HK board for this build if possible, will re-flash the firmware and try again, would erasing the HK board before re-flashing help and if so how is this done?

Found some instructions for erasing it in Arduino, but from mine I can't find the eprom clear in the examples file :(

Post edit:

Re-installing the firmware didn't help, installed V2.8 R3, now everything working?

**Comment by edw00d4711** [comment]
https://code.google.com/p/arducopter/wiki/AC2_ESC

If you already did that:
Was your throttle set to zero when you connected the lipo?

**Comment by Kmonk** [comment]
Yes, I think the problem was with V3.0.2 R2 and the HK AIO pro V2 board, as it worked fine with V2.8 R3.
